apps/mobile-host/ios/fastlane/README.md

```diff
@@ -0,0 +1,40 @@
+fastlane documentation
+----
+
+# Installation
+
+Make sure you have the latest version of the Xcode command line tools installed:
+
+```sh
+xcode-select --install
+```
+
+For _fastlane_ installation instructions, see [Installing _fastlane_](https://docs.fastlane.tools/#installing-fastlane)
+
+# Available Actions
+
+## iOS
+
+### ios custom_lane
+
+```sh
+[bundle exec] fastlane ios custom_lane
+```
+
+Description of what the lane does
+
+### ios tf_dist
+
+```sh
+[bundle exec] fastlane ios tf_dist
+```
+
+Builds and uploads the app to TestFlight
```

Comment:
Let's remove `custom_lane` and `tf_dist` actions from there and describe valid `testflight_distribution ` lane instead